src/usr/local/www/vpn_ipsec_mobile.php

```diff
@@ -252,7 +266,13 @@
 		$client['group_source'] = $pconfig['group_source'];
 
 		if ($pconfig['pool_enable']) {
-			$client['pool_address'] = $pconfig['pool_address'];
+			if($pconfig['radius_ip_priority_enable'])
+			{
+				$client['pool_address'] = '%radius,' . $pconfig['pool_address'];
```

Comment:
This is a rather awkward way to do this. Normally you'd check the value of `radius_ip_priority_enable` in the backend code every place that uses `pool_address` and add it there, rather than having it added in the GUI field directly.